gatsby/content/projects/bridges/mx-puppet-discord.mdx

```diff
@@ -25,15 +25,16 @@ bridges: Discord
 &#10003; Formatted Messages  
 &#10003; Media/files  
 &#10003; Redactions/Deletions  
-&#10003; Presence  (Discord &#8594; Matrix only)  
-&#10003; Typing notifications (Discord &#8594; Matrix only)  
+&#10003; Presence  (both ways)  
+&#10003; Typing notifications (both ways)  
 &#10003; Replies  
 &#10003; Edits  
 &#10003; Reactions  
 &#10003; Stickers  
 &#10003; Multi-user  
 &#10003; Friends management  
 &#10003; Initiate chats from matrix  
-&#10007; History  
+&#10007; History   
+&#10007; Threads  
```

Comment:
Threads support surprises me as the [issue #164](https://github.com/matrix-discord/mx-puppet-discord/issues/164) is still open

Reply:
@MRAAGH What's the reality here?

Reply:
Threads are not bridged.
Notice that I added it as "✗", not "✓".
This was missing because the page has not yet been updated since the time threads were introduced to Discord. So I added it.